### PR TITLE
feat: add Bun install on existing project

### PR DESCRIPTION
**Pull request by PlayerNguyen**
Add Bun install on an existing project with `bun install` command as an additional step.

### CODE REVIEW

**Comment by ryuapp** [comment]
Hello.
The purpose of getting started is to create a new Hono project, so this addition seems unnecessary.
The guide should be simple.

However, your point is correct, so why don't u add it as an other section(add Hono to an exsisting project)?

**Comment by PlayerNguyen** [comment]
Hi @ryuapp, 

I understand that we can put this into a different section. However, I am not too familiar with this documentation format. Moreover, this one, I expected worked and tested with `Bun`. So the only question here is "It is necessary to create a new section instead?".

**Comment by PlayerNguyen** [comment]
I found that the [Getting Started > Basic ](https://hono.dev/getting-started/basic) is quite suitable for putting the section above.

File: getting-started/bun.md [hunk 1]
```diff
@@ -23,6 +24,13 @@ cd my-app
 bun install
 ```
 
+### 2.2. Setup an existing project
+On an existing Bun project, we only need to install `hono` dependencies on the project root directory via
+
+```sh
+bun install hono
```
Comment (yusukebe) [inline getting-started/bun.md:31]: I think [`bun add` ](https://bun.sh/guides/install/add) is better. What do you think of it?

**Comment by yusukebe** [comment]
Hi @PlayerNguyen !

This is a good suggestion. I've left a comment. Check it.

File: getting-started/bun.md (hunk 1 above)
Comment (PlayerNguyen) [reply getting-started/bun.md:31]: Sure, I am going to change it.

**Comment by PlayerNguyen** [comment]
@yusukebe I already use `bun add` instead. Can you please check it again?

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@PlayerNguyen Looks good! Merging now. Thank you for your contribution!

**Pull request merged**